src/NUnitConsole/nunit-console/nunit-console.csproj

```diff
@@ -35,6 +35,8 @@
     <DefineConstants>TRACE;NUNIT_CONSOLE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Commandlineparameters>nunit.engine.tests.dll -process:Single</Commandlineparameters>
```

Comment:
We should probably remove these at some point. They belong in the .user files for most projects and just create noise when either of us changes the debugging settings.